### DISCUSSION TITLE
Documentation Site

### DISCUSSION THREAD

**Discussion by vrajroham** [vrajroham/laravel-bitpay#37] [Ideas]
Need suggestions for creating the documentation site for Laravel Bitpay

**Comment by alexstewartja** [comment]
@vrajroham The same thought crossed my mind while updating the README for #35. The suggestions I have so far, in order of fastest turnaround time are:

- [GH Wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis) (ex. https://github.com/vrajroham/laravel-bitpay/wiki)
- [GH Pages](https://pages.github.com/) with a custom domain (ex. https://laravelbitpay.vrajroham.me)
- [RTD](https://readthedocs.org/)

**Reply by alexstewartja** [reply]
So most definitely Docusaurus: https://docusaurus.io/docs
